packages/cli/src/commands/init.ts

```diff
@@ -80,6 +92,11 @@ async function handleOnboardingFlow(config: ConfigService) {
 
     const envAnswer = await prompt(environmentQuestions);
     if (envAnswer.env === 'self-hosted-docker') {
+      analytics.track({
+        anonymousId,
+        event: 'Self-hosted Docker',
```

Comment:
If it is technically possible because our analytics service allows to it, I would highly recommend the events to be in a shape of objects, where we would have an event type (to distinguish events like you are doing by name right now) and where we could add in the future relevant information useful for the analytics.
Example:
```
interface AnalyticsEvent {
   type: AnalyticsEventType;
   ...
   // And maybe in the future we can add things like... as needed
   _organizationId: string;
   _environmentId: string;
   data: Record<string, unknown>;
   ...
}
```

Reply:
Segment have an extra object for `traits` or custom data. So the event is only string and optional data can be passed down.

Reply:
Ah good we have the traits implementation. Then I would enum the different events as extra.